charts/k8s-monitoring/test/query-test.sh

```diff
@@ -51,10 +137,13 @@ count=$(jq -r ".queries | length-1" "${1}")
 for i in $(seq 0 "${count}"); do
   query=$(jq -r --argjson i "${i}" '.queries[$i].query' "${1}")
   type=$(jq -r --argjson i "${i}" '.queries[$i] | .type // "promql"' "${1}")
+  expectedCount=$(jq -r --argjson i "${i}" '.queries[$i].expect.count // empty | tostring' "${1}")
+  expectedValue=$(jq -r --argjson i "${i}" '.queries[$i].expect.value // empty | tostring' "${1}")
+  expectedOperator=$(jq -r --argjson i "${i}" '.queries[$i].expect | .operator // "=="' "${1}")
```

Comment:
I wonder if we should/could do the query testing in a different language (like Go) for longevity sake. Something to think about.

Reply:
Perhaps.
A reason I like Bash for this is that I have run this script directly before. Just export some environment vars and go.
It makes things very portable without having to compile it.

Reply:
Sure, that makes sense